### PR TITLE
Feature/templateengine First version of template engine for iofog 

### PR DESCRIPTION
**Pull request by franckOL**
### Description
Description
Early stage development for template engine based on liquidjs. see franckOL#1 .

The first version has only a self object to help configuration, and it refere to the document itself.
The template engine liquidjs is used for all POST, PUT verbs for all API routes.

Next improvements:
- [ ] only apply on precise route: routes use by agent don't need that
- [ ]  Add all controller applications to context of template rendering (lazy loading)
- [ ] Add all controller edge ressources to context of template rendering (lazy loading)
- [ ] Add all controller agent to context of template rendering (lazy loading)

### CODE REVIEW

**Comment by franckOL** [comment]
Can't fix legal agreements

File: src/helpers/template-helper.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+ *  * This program and the accompanying materials are made available under the
+ *  * terms of the Eclipse Public License v. 2.0 which is available at
+ *  * http://www.eclipse.org/legal/epl-2.0
+ *  *
+ *  * SPDX-License-Identifier: EPL-2.0
+ *  *******************************************************************************
+ *
+ */
+// ninja2 like template engine
+var { Liquid } = require('liquidjs')
```
Comment (Pixcell) [inline src/helpers/template-helper.js:14]: This should be `const`, not `var`

File: src/helpers/template-helper.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+let templateEngine = new Liquid()
```
Comment (Pixcell) [inline src/helpers/template-helper.js:15]: Why use `let` instead of `const`, the value of `templateEngine` never gets reassigned later on

**Review by Pixcell — CHANGES_REQUESTED** [review]
Almost ready to be approved. Two minor changes to use const instead of var or let, and we should be good to go.

It seems like snyk step of the pipeline is failing. This might be due to misconfiguration.
Can you disable the step at the moment: https://github.com/eclipse-iofog/Controller/blob/8d1685aef7ec2fde01021272ca0a95abb6a90e4f/azure-pipelines.yml#L48 (Comment out that step in the file)

File: src/helpers/template-helper.js (hunk 2 above)
Comment (franckOL) [reply src/helpers/template-helper.js:15]: OK. I forget this two, npm run standard does not shown me the problem.

**Comment by franckOL** [comment]
We can close I redo the pull request

**Pull request closed without merging**